user-data-vault.sh

```diff
@@ -1,25 +1,43 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
If this is a template, it should be marked accordingly, by using the `.tpl` extension, as documented here: https://www.terraform.io/docs/providers/template/